app/lib/efile/nj/nj1040_calculator.rb

```diff
@@ -147,6 +147,14 @@ def calculate_line_6
         number_of_line_6_exemptions * 1_000
       end
 
+      def line_7_self_checkbox
+        Efile::Nj::NjSenior.is_over_65(@intake.primary_birth_date)
```

Comment:
[DUST] Cool to see this logic broken out!

Reply:
ended up using `@intake.primary_senior?` and `@intake.spouse_senior?` after all